.editorconfig

```
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
```

```suggestion
indent_size = 4
```

// Ведь не мне одному кажется, что с четырьмя пробелами читать намного легче, правда?)

---

Мне придётся привыкать, я в таких больших отступах расплываюсь.. может быть у тебя все буковы настолько мелкие, что и отступы не такие большие? =))
Пока поставлю 4... попробую.